packages/experience-builder-sdk/src/blocks/VisualEditorContext.tsx

```diff
@@ -187,16 +187,11 @@ export function VisualEditorContextProvider({
           }
           break;
         }
-        case INCOMING_EVENTS.SelectedComponentChanged: {
```

Comment:
Are you sure we don't need it?

It looks like we send this event when [bound node is updated with changes via slide in editor](https://github.com/contentful/user_interface/blob/master/src/javascripts/features/experience-builder/contexts/useCompositionCanvasSubscriber.ts#L301)

Reply:
I am removing it here: https://github.com/contentful/user_interface/pull/18193/files we have two events for the same purpose. The user_interface PR will need to go first and I've tested it with v2.6.2 sdk version and I didn't see any issues

Reply:
Ah, I see it now. Thanks for pointing it out